agent/bench-scripts/pbench-run-benchmark.pl

```diff
@@ -317,10 +319,6 @@ sub find_common_parameters {
         push(@iterations_labels, $iteration_label);
     }
 
-    if (! $pp_only) {
-    	system("pbench-init-tools --group=" . $tool_group . " --dir=" . $base_bench_dir);
```

Comment:
Why are these statements being removed?

Reply:
The commit log message says:

>The pbench-init-tools and pbench-end-tools were replaced with the
tool-master transition and are not needed in pbench-run-benchmark
anymore as the tool master is handled in pbench-run-benchmark-sample.

where I assume "tool-master" should be "tool-meister"....  🙂

Reply:
That's right, `pbench-tool-meister-start` should handle all that.

Reply:
@ldoktor, note that this change needs to be reflected in changes to the "gold" files:
- `agent/bench-scripts/tests/pbench-run-benchmark/test-44.txt`
- `agent/bench-scripts/tests/pbench-run-benchmark/test-45.txt`
- `agent/bench-scripts/tests/pbench-run-benchmark/test-46.txt`
- `agent/bench-scripts/tests/pbench-run-benchmark/test-47.txt`

Reply:
First I tried replacing them with `pbench-tool-meister-start` but that resulted in double execution. But honestly I couldn't find where it's being initialized, it just is (the pre sysinfo is present and TM log file shows the same sequence as for pbench-fio.